doc/contributing/pull-requests.md

```diff
@@ -184,6 +184,10 @@ A good commit message should describe what changed and why.
    of the log. Use the `Fixes:` prefix and the full issue URL. For other
    references use `Refs:`.
 
+   `Fixes:` references get automatically added to your commit when the PR
+   is merged, so it is sufficient to just link your PR with the issue or issues it is
+   fixing directly on GitHub (for more details see the [GitHub documentation](https://docs.github.com/en/issues/tracking-your-work-with-issues/using-issues/linking-a-pull-request-to-an-issue))
```

Comment:
This link should be added to the end of the document just like in line 170

Reply:
Yes, my bad, sorry, I've fixed that 🙇